umpleonline/scripts/codemirror6/editor.mjs

```diff
@@ -102,41 +109,17 @@ const changeListenerPlugin = ViewPlugin.fromClass(class {
         
         if (newContent !== this.lastContent) {
         const currentPositionofCursor = this.view.state.selection.main.head;
-       // if (newContent.localeCompare(this.lastContent) !== 0) {
-
-        if (debuggerFlag)   
-       console.warn('Content changed');
-
-        //console.warn('new content:', newContent); 
-        if (debuggerFlag)
-        console.log('new content lenght:', newContent.length);
 
-        // console.warn('old content:', this.lastContent);
-        if (debuggerFlag)
-        console.log('old content lenght:', this.lastContent.length);
+        if (debuggerFlag) {
```

Comment:
We can name this as **codemirrorDebuggerFlag**

Reply:
sure, I will do these changes today